### PR TITLE
test: ActionsTests improvements

### PR DESCRIPTION
**Pull request by Dor-bl**
Added SetUp and TearDown Attributes
Moved the active element text one step earlier for easier debugging

### CODE REVIEW

File: src/FlaUI.WebDriver.UITests/ActionsTest.cs [hunk 1]
```diff
@@ -9,32 +9,44 @@ namespace FlaUI.WebDriver.UITests
...
+        public void Setup()
+        {
+            var driverOptions = FlaUIDriverOptions.TestApp();
+            driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
+        }
+
+        [TearDown]
+        public void Teardown()
+        {
+            driver?.Quit();
```
Comment (aristotelos) [inline src/FlaUI.WebDriver.UITests/ActionsTest.cs:24]:
```suggestion
            driver?.Dispose();
```

File: src/FlaUI.WebDriver.UITests/ActionsTest.cs [hunk 2]
```diff
@@ -9,32 +9,44 @@ namespace FlaUI.WebDriver.UITests
     [TestFixture]
     public class ActionsTests
     {
+        private RemoteWebDriver driver;
```
Comment (aristotelos) [inline src/FlaUI.WebDriver.UITests/ActionsTest.cs:12]: I'd like to comply with https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md, so this should be:
```suggestion
        private RemoteWebDriver _driver;
```

**Review by aristotelos — CHANGES_REQUESTED** [review]
Added a few comments, thanks for the contribution!

**Review by aristotelos — COMMENTED** [review]
The use of `_driver` on line 12 and `driver` on line 18 won't build...

**Comment by Dor-bl** [comment]
> The use of `_driver` on line 12 and `driver` on line 18 won't build...

Right..wasn't paying attention

**Comment by Dor-bl** [comment]
@aristotelos fixed

**Review by aristotelos — APPROVED** [review]

**Pull request merged**